src/main/frontend/JS/Admin.js

```diff
@@ -0,0 +1,405 @@
+            }
+            // Assuming the response is a file, trigger the download
+            return response.blob();
+        })
+        .then(blob => {
+            const url = window.URL.createObjectURL(blob);
+            const a = document.createElement('a');
+            a.href = url;
+            a.download = 'scenefile.scn';
+            document.body.appendChild(a);
+            a.click();
+            document.body.removeChild(a);
+            window.URL.revokeObjectURL(url);
+        })
+        .catch(error => {
+            console.error('Error downloading scene file:', error.message);
+        });
+}
+
+
+function uploadFile() {
+    const formData = new FormData(document.getElementById('uploadForm'));
+    const url = 'http://127.0.0.1:5000/scene/upload'
+    fetch(url,
+    {
+            method: 'POST',
+            body: formData
+        }).then((value) => {
+            fetchData(pagenumber, pagesize, sortOrder)
+        }
+    );
+// Clear file input fields after upload
+    document.getElementById("Flights").value = ''
+    document.getElementById("Landings").value = ''
+}
+
+
+function pageup() {
+    if (pagenumber < rowcount) {
+        pagenumber = pagenumber + 1;
+        fetchData(pagenumber, pagesize, sortOrder);
+    }
+}
+
+function pagemax() {
+    pagenumber = rowcount;
+    fetchData(pagenumber, pagesize, sortOrder);
+    document.getElementById("pagemax").disabled = true
+}
+
+function pagemin() {
+    pagenumber = 1;
+    fetchData(pagenumber, pagesize, sortOrder);
+    document.getElementById("pagemin").disabled = true
+}
+
+function pagedown() {
+    if (pagenumber > 1) {
+        pagenumber = pagenumber - 1;
+        fetchData(pagenumber, pagesize, sortOrder);
+    }
+}
+
+
+async function fetchData(pageNumber, pageSize, order) {
+    try {
+        const searchValue = document.getElementById("search").value;
+        let url = `http://127.0.0.1:5000/scene/complete/table?` +
+            `pageNumber=${pageNumber}&` +
+            `pageSize=${pageSize}&` +
+            `sortBy=${cellContent}&` +
+            `order=${order}&` +
+            `searchfilter=${searchValue}`;
+        const response = await fetch(url);
+
+        if (!response.ok) {
+            throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+        const responseData = await response.json();
+
+        // Call your function to update the table with the response data
+        processData(responseData.data);
+        rowcount = responseData.pagenumber + 1; // Update totalPages variable
+        document.getElementById("pagemax").disabled = pagenumber >= rowcount;
+        document.getElementById("pagemin").disabled = pagenumber <= 1;
+        if (rowcount < 1) {
+            rowcount = 1;
+        }
+        page.innerText = "Page: " + pagenumber + " of " + rowcount;
+    } catch (error) {
+        console.error('Error fetching paginated data:', error.message);
+    }
+}
+
+let cellContent
+if (!cellContent) {
```

Comment:
this just seems wierd to me? Could you explain why you did it this way?

Reply:
cellContent is waar ik op sort, deze word ergens anders geset. al sort ik nergens op wil ik dat cellContent FLIGHT_ID is